app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ActiveRecord::Base
+  belongs_to :author, class_name: 'User', foreign_key: :user_id
+  has_many :likes
+  has_many :comments
+
+  def five_last_comments
+    comments.order('created_at Desc').limit(5)
+  end
+
+  after_save :update_posts_counter
+
+  private
+
+  def update_posts_counter
+    author.increment!(:posts_counter)
```

Comment:
- [ ] Kindly change this `author` on line 15 to `user` because you do not have `author` in your database. Line 15 should look like this `user.increment!(:PostsCounter)`

Reply:
This has been resolved.